src/igvfd/upgrade/treatment.py

```diff
@@ -10,3 +10,12 @@ def treatment_1_2(value, system):
     if 'documents' in value:
         if len(value['documents']) == 0:
             del value['documents']
+
+
+@upgrade_step('treatment', '2', '3')
+def treatment_2_3(value, system):
+    # https://igvf.atlassian.net/browse/IGVF-292
+    if 'purpose' not in value:
+        value['purpose'] = 'perturbation'
+        value['notes'] = 'This treatment did not have purpose specified previously, it was upgraded to have perturbation purpose.'
```

Comment:
@khzlin you can not assume the notes field was empty, so need to concatenate rather than overwrite it